safe_autonomy_sims/gym/inspection/reward.py

```diff
@@ -7,7 +7,7 @@
 from safe_autonomy_sims.pettingzoo.inspection.utils import delta_v, rel_dist
 
 
-def observed_points_reward(chief: sim.Target, prev_num_inspected: int) -> float:
+def observed_points_reward(chief: sim.Target, prev_num_inspected: int, scale: float = 0.01) -> float:
```

Comment:
Update docstring to include scale argument

Reply:
sure thing! I updated the docstrings for the observed points reward. I also went ahead and updated the docstrings for the delta v reward and util functions. I'm waiting on updating docstrings for the environments until the changes there are solidified.